account_financial_report/report/open_items.py

```diff
@@ -638,6 +639,7 @@ def _compute_partners_and_accounts_cumul(self):
         self._compute_account_cumul()
 
     def _compute_partner_cumul(self):
+        # pylint: disable=sql-injection
```

Comment:
To avoid disabling pylint, I think here we should remove variables like `where_condition_partner_by_account` and write the string directly at the end of `query_computer_partner_residual_cumul` and `query_compute_partners_due_cumul`, right?

Reply:
Well, I haven't seen full code right now, but you know the basis. Do what you consider best :smiley:

Reply:
Done